gravitee-am-ui/src/app/domain/settings/password-policy/domain-password-policy.component.html

```diff
@@ -0,0 +1,96 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div>
+  <div>
+    <div fxFlex="70">
+      <form (ngSubmit)="update()" (keydown.enter)="false" fxLayout="column" #applicationForm="ngForm">
+        <div *ngIf="editMode">
+          <div class="gv-form-section">
+            <h4>Policy Type</h4>
+            <mat-form-field appearance="outline" floatLabel="always">
+              <mat-select [(value)]="selectedPolicyType" (selectionChange)="formChange()"
+                          placeholder="Password policy type">
+                <mat-option value="regex"><em>Regex</em></mat-option>
+                <mat-option value="policyDetails"><em>Policy details</em></mat-option>
+              </mat-select>
+            </mat-form-field>
+          </div>
+
+          <div class="gv-form-section" *ngIf="selectedPolicyType == 'regex'">
+            <h4>Regex format</h4>
+            <mat-form-field appearance="outline" floatLabel="always">
+              <input matInput type="text" placeholder="Enter the regex format" name="regexFormat"
+                     [(ngModel)]="regexFormat">
+            </mat-form-field>
+          </div>
+
+          <div class="gv-form-section" *ngIf="selectedPolicyType != 'regex'">
+            <h4>Minimum length</h4>
+            <mat-form-field appearance="outline" floatLabel="always">
+              <mat-select [(value)]="minLength" (selectionChange)="formChange()">
+                <mat-option [value]=4><em>4</em></mat-option>
+                <mat-option [value]=6><em>6</em></mat-option>
+              </mat-select>
+            </mat-form-field>
+            <h4>Max length</h4>
```

Comment:
By looking to others solutions and some UX/UI design, I think we can remove the max length option (should be fixed and defined by OWASP recommendation).

Reply:
ok